tests/subsys/settings/fcb/src/settings_test_fcb.c

```diff
@@ -341,66 +341,50 @@ int c3_handle_export(int (*cb)(const char *name,
 	return 0;
 }
 
-void test_settings_check_target(void)
+void *settings_config_fcb_setup(void)
 {
 	const struct flash_area *fap;
 	int rc;
 	uint8_t wbs;
 
 	rc = flash_area_open(FLASH_AREA_ID(storage), &fap);
-	zassert_true(rc == 0, "Can't open storage flash area");
+	zassume_true(rc == 0, "Can't open storage flash area");
```

Comment:
It is okay to have this be a zassume, but I just wanted to check that you understand this can be a zassert now.

Reply:
Thank you, got it.